cypress/support/onboardingCommands.ts

```diff
@@ -7,34 +7,34 @@ declare global {
        * @description Start onboarding mnemonic wallet process
        * @param {string[]} recoveryWords - pass an empty array to be filled later
        */
```

Comment:
why is this change needed? @fullstackninja864

Reply:
These are just lint changes, which was throwing error/warning as we have added lint for test cases too

Reply:
In order to support the dark and light mode i had updated this file and then after before creating commit I have to fix them to pass pre-commit hooks.